packages/rich-text/src/component/use-input-and-selection.js

```diff
@@ -275,29 +275,10 @@ export function useInputAndSelection( props ) {
 				return;
 			}
 
-			if ( ! isSelected ) {
-				// We know for certain that on focus, the old selection is invalid.
-				// It will be recalculated on the next mouseup, keyup, or touchend
-				// event.
-				const index = undefined;
-
-				record.current = {
-					...record.current,
-					start: index,
-					end: index,
-					activeFormats: EMPTY_ACTIVE_FORMATS,
-				};
-				onSelectionChange( index, index );
```

Comment:
The main goal for my explorations on this PR is to get rid of this, which was causing onSelectionChange to be called twice on focus, I'm assuming it's already done in the "selectionchange" event.

Reply:
If it's called twice, it will be called first here, then in the "selectionchange" event.